qp_deblur/deblur.py

```diff
@@ -129,14 +131,31 @@ def deblur(qclient, job_id, parameters, out_dir):
     # Generating artifact
     pb = partial(join, out_dir)
 
+    # Generate the filepaths
+    final_biom = pb('final.biom')
+    final_seqs = pb('final.seqs.fa')
+    final_biom_16s = pb('final.only-16s.biom')
+    final_seqs_na = pb('final.seqs.fa.no_artifacts')
+
+    if not exists(final_biom_16s):
+        # Create an empty table. We need to send something to Qiita that is
+        # a valid BIOM, so we are going to create an empty table
+        t = Table([], [], [])
+        with biom_open(final_biom_16s, 'r+') as f:
```

Comment:
Interesting, don't we want to just write the file? I almost want to say that we want to do w-.

This is a minor digression, shouldn't be a blocker to merge if someone else approves this PR.